### PR TITLE
Upgrade to Storybook 7

### PR DESCRIPTION
**Pull request by Nateowami**
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/web-xforge/1786)

### CODE REVIEW

File: src/SIL.XForge.Scripture/ClientApp/src/app/shared/notice/notice.stories.ts [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import { CommonModule } from '@angular/common';
-import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
-import { Meta, Story } from '@storybook/angular';
+import { Meta, StoryFn } from '@storybook/angular';
 import { NoticeComponent } from 'src/app/shared/notice/notice.component';
```
Comment (pmachapman) [inline src/SIL.XForge.Scripture/ClientApp/src/app/shared/notice/notice.stories.ts:3]: Nit: Should this be `../../` instead of `src/app`?

**Review by pmachapman — APPROVED** [review]
Thank you for this PR - it fixes about 21 vulnerabilities reported via `npm audit`, and Storybook works great via `npm run storybook` on my laptop.

A couple of minor things which I don't see as blocking, but thought I might raise in case you thought they need changing:
 * I noticed a nit where an import was `src/app` instead of `../..`
 * The imports in the storybook TS files were of often out of order. Lint doesn't report these as issues, so I don't think they are a big deal.
 * Storybook 7.0.6 was released a few days ago - we could update to this version in this PR if you want to?

**Review by Nateowami — COMMENTED** [review]
> The imports in the storybook TS files were of often out of order. Lint doesn't report these as issues, so I don't think they are a big deal.

This would probably indicate the linter isn't running, which is a lot more concerning than the actual import issues. But as far as I can tell it is running; when I changed `src/app` to `../..` it complained about the import order and made me fix it.

> Storybook 7.0.6 was released a few days ago - we could update to this version in this PR if you want to?

Thanks. Updated.

> *[Reviewable](https://reviewable.io/reviews/sillsdev/web-xforge/1786#-:-NTk5KoRCVtqQWRKYP3H:b1ksepl)* status: 0 of 13 files reviewed, all discussions resolved (waiting on @pmachapman)

___
*[`src/SIL.XForge.Scripture/ClientApp/src/app/shared/notice/notice.stories.ts` line 3 at r1](https://reviewable.io/reviews/sillsdev/web-xforge/1786#gh-1174653528:-NTk5NwPEU2BIav0P7ww:b-896fix) ([raw file](https://github.com/sillsdev/web-xforge/blob/6077052938bf833bb401232ace104b1e0a56e1d7/src/SIL.XForge.Scripture/ClientApp/src/app/shared/notice/notice.stories.ts#L3)):*
<details><summary><i>Previously, pmachapman (Peter Chapman) wrote…</i></summary><blockquote>

Nit: Should this be `../../` instead of `src/app`?
</blockquote></details>

Done.

**Review by pmachapman — APPROVED** [review]
> Reviewed 10 of 13 files at r1, 3 of 3 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/web-xforge/1786)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @Nateowami)

**Review by pmachapman — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/sillsdev/web-xforge/1786#-:-NTkAveh7Y9kVq5ep43P:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @Nateowami)

**Pull request merged**